tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -152,9 +155,24 @@ oe_result_t gen_report(oe_enclave_t* enclave)
                 NULL, remote_report, report_size, &parsed_report);
             if (result != OE_OK)
             {
-                log("Failed to verify report. result=%u (%s)\n",
+                printf(
```

Comment:
These statements should use `log()` instead of `printf()`, otherwise it bypasses the logging to file functionality provided by oecertdump.